src/Main.cc

```diff
@@ -522,29 +531,46 @@ int main(int argc, const char* argv[]) {
         int port(3002);
         int thread_count = TBB_THREAD_COUNT;
         int omp_thread_count = OMP_THREAD_COUNT;
+        string frontend_folder;
+        string host;
+        bool no_http = false;
+        bool debug_no_auth = false;
+        bool no_browser = false;
+
         { // get values then let Input go out of scope
             casacore::Input inp;
             inp.version(VERSION_ID);
             inp.create("verbose", "False", "display verbose logging", "Bool");
             inp.create("perflog", "False", "display performance logging", "Bool");
+            inp.create("no_http", "False", "disable CARTA frontend HTTP server", "Bool");
+            inp.create("debug_no_auth", "False", "accept all incoming WebSocket connections (insecure, use with caution!)", "Bool");
+            inp.create("no_browser", "False", "prevent the frontend from automatically opening in the default browser on startup", "Bool");
+            inp.create("host", "", "only listen on the specified interface");
```

Comment:
If this restriction only applies to serving the frontend, should the name and/or description reflect that?